fem/fe.cpp

```diff
@@ -7176,6 +7177,18 @@ PositiveTensorFiniteElement::PositiveTensorFiniteElement(
                            dims > 1 ? FunctionSpace::Qk : FunctionSpace::Pk),
      TensorBasisElement(dims, p, BasisType::Positive, dmtype) { }
 
+VectorTensorFiniteElement::VectorTensorFiniteElement(const int dims,
```

Comment:
I don't think `TensorBasisElement` is correct in that case, as suggested earlier I think we should create something like `MixedVectorTensorBasisElement`.